db/migrate/20170821152307_add_source_type_to_permission_templates.rb

```diff
@@ -1,7 +1,16 @@
 class AddSourceTypeToPermissionTemplates < ActiveRecord::Migration[4.2]
   def change
     add_column :permission_templates, :source_type, :string
-    rename_column :permission_templates, :admin_set_id, :source_id
+    add_column :permission_templates, :source_id, :string
+
+    Hyrax::PermissionTemplate.find_each do |permission_template|
+      permission_template.source_type = 'admin_set'
```

Comment:
My understanding is that this should usually be in class name format (e.g. "AdminSet") as we will often constantize the string (`permission_template.source_type.constantize` would return the class `AdminSet`).

Reply:
Presently there seems to be other code that makes use of 'admin_set' for source_type. Not saying your proposal is not valid, it may just be a separate Issue?  https://github.com/samvera/hyrax/blob/collections-sprint/app/controllers/hyrax/admin/permission_template_accesses_controller.rb#L31

Reply:
Two thoughts here...
* The use of `if source_type == 'admin_set'` was brought in by the addition of the source_type column, so we can certainly change this to define source_type as either Collection or AdminSet.
* Use of admin_set and collection does not prevent use of constantize...
```
>> 'admin_set'.camelize.constantize
=> AdminSet
>> 'collection'.camelize.constantize
=> Collection
```
@jeremyf Considering these observations, does that lean you strongly in one direction or another.  Or is this minor.  It will take a bit of refactoring to change all the places where we are currently checking source_type.  That said, if you think this will be a problem down stream, it is better to bite that bullet now rather than later.